core/src/main/scala/com/softwaremill/events/EventMachine.scala

```diff
@@ -45,6 +46,24 @@ class EventMachine(
       })
   }
 
+  def failOverFromStoredEventsTo(timeLimit: OffsetDateTime = OffsetDateTime.now()): Unit = {
```

Comment:
I'd take as a param a `Clock`, so that you can easily substitute time for testing